### PR TITLE
[BUG] fix language auto detect

### PR DESCRIPTION
**Pull request by BubbleBear**
## What are the changes?
currently language auto detection is broken,
fixed according to [i18next documentation](https://www.i18next.com/how-to/faq#how-should-the-language-codes-be-formatted)

## Why am I doing these changes?
currently language auto detection is broken for **Simplified Chinese, Traditional Chinese, Portugese Brazilian**

## What did change?
i18next config resource key

#### BEFORE FIX
image

----

#### AFTER FIX
image

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Comment by Tempo-anon** [comment]
Can you show some proof that this works?

File: src/plugins/i18n.ts [hunk 1]
```diff
@@ -174,13 +174,13 @@ export function initI18n(): void {
...
-      pt_BR: {
+      "pt-BR": {
         ...ptBrConfig
       },
-      zh_CN: {
+      "zh-CN": {
         ...zhCnConfig
       },
-      zh_TW: {
+      "zh-TW": {
```
Comment (bennybroseph) [inline src/plugins/i18n.ts:183]: assuming the rest is correct, this adds quotes here which cannot be correct

**Review by bennybroseph — CHANGES_REQUESTED** [review]

File: src/plugins/i18n.ts (hunk 1 above)
Comment (BubbleBear) [reply src/plugins/i18n.ts:183]: this is the grammer, you cannot use `-` in object key unless wrap it in quotes

**Comment by BubbleBear** [comment]
> Can you show some proof that this works?

sure thing sir

**Comment by BubbleBear** [comment]
@Tempo-anon @bennybroseph 
Thanks for your review and patiences, I've added some screenshot proofs.

**Comment by davidyan-hub** [comment]
I did not have envirement to do this...
Who can modify the text.ts to change the font-family and font-size for chinese.

Code as below:

fontFamily: localStorage.getItem('prLang')==="zh_CN"?'simhei':'emerald',

styleOptions.fontFamily = localStorage.getItem('prLang')==="zh_CN"?'MuZaiPixel':'pkmnems';

and same judgement logic for changing font-size, let the size is smaller.

**Review by bennybroseph — APPROVED** [review]

**Comment by BubbleBear** [comment]
> I did not have envirement to do this... Who can modify the text.ts to change the font-family and font-size for chinese.
> 
> Code as below:
> 
> fontFamily: localStorage.getItem('prLang')==="zh_CN"?'simhei':'emerald',
> 
> styleOptions.fontFamily = localStorage.getItem('prLang')==="zh_CN"?'MuZaiPixel':'pkmnems';
> 
> and same judgement logic for changing font-size, let the size is smaller.

it's not like that easy, i've tried but the font size logic runs different from CSS

**Pull request merged**